### PR TITLE
[ACA-2983] Changed the labels from the inputs name and type to always floating

### PR DESCRIPTION
**Pull request by ursedaniel**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [x] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://issues.alfresco.com/jira/browse/ACA-2983
The labels from creating a new process (process name and type) are in the same position as the place holder.

**What is the new behaviour?**

Added floating labels from the 2 inputs and switched the positions of the 2 inputs.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services/src/lib/process-list/components/start-process.component.scss [hunk 1]
```diff
@@ -14,13 +14,19 @@
             width: 100%;
             padding: 16px 0 0;
         }
+        .mat-form-field-label {
+            color: #bababa;
```
Comment (DenysVuika) [inline lib/process-services/src/lib/process-list/components/start-process.component.scss:18]: this should be taken from the theme

File: lib/process-services/src/lib/process-list/components/start-process.component.scss [hunk 2]
```diff
@@ -1,54 +1,61 @@
...
-            text-align: right;
-        }
-        .mat-button {
-            text-transform: uppercase !important;
-        }
-    }
-    &-process-input-container {
-        mat-form-field {
-            width: 100%;
+@mixin adf-process-services-create-theme($theme) {
```
Comment (DenysVuika) [inline lib/process-services/src/lib/process-list/components/start-process.component.scss:1]: If you change the style to mixin, it should be called where other mixins are, and also component should not reference it in the styles attribute.

Comment (DenysVuika) [reply lib/process-services/src/lib/process-list/components/start-process.component.scss:1]: check the _index.scss for examples

Comment (ursedaniel) [reply lib/process-services/src/lib/process-list/components/start-process.component.scss:1]: sorry forgot to commit the file for import

Comment (DenysVuika) [reply lib/process-services/src/lib/process-list/components/start-process.component.scss:1]: also, the .ts file should not contain the following line now:
```
styleUrls: ['./start-form.component.scss'],
```

File: lib/process-services/src/lib/process-list/components/start-process.component.scss [hunk 3]
```diff
@@ -1,54 +1,61 @@
...
+            }
+            mat-form-field {
+                width: 100%;
+            }
+            mat-select {
+                width: 100%;
+                padding: 16px 0 0;
+            }
+            .mat-form-field-label {
+                color: map_get($mat-grey,A400);
```
Comment (DenysVuika) [inline lib/process-services/src/lib/process-list/components/start-process.component.scss:19]: I think this is not correct, we are using "map-get" in other files, see 
https://github.com/Alfresco/alfresco-ng2-components/blob/development/lib/content-services/src/lib/breadcrumb/breadcrumb.component.scss#L18

File: lib/process-services/src/lib/process-list/components/start-process.component.scss [hunk 4]
```diff
@@ -1,54 +1,61 @@
...
+    .adf {
+        &-start-process {
+            width: 96%;
+            margin-left: auto;
+            margin-right: auto;
+            margin-top: 10px;
+            .mat-select-trigger {
+                font-size: 14px !important;
+            }
+            mat-form-field {
```
Comment (eromano) [inline lib/process-services/src/lib/process-list/components/start-process.component.scss:11]: do we need to overwrite all this mat- class?

Comment (ursedaniel) [reply lib/process-services/src/lib/process-list/components/start-process.component.scss:11]: Can you be more explicit about what exact lines of code are you reffering to?

**Pull request merged**